dev-tools/command-line-advanced.md

```diff
@@ -99,7 +99,7 @@ Front-end development is web development, and the web has a strong bias towards
 - Search with `grep`
 
   - Show files containing the word important in this directory
-    ```bash
+    ```
```

Comment:
Removing these gets rid of minor highlighting. It would be better to change them to `sh`, which is agnostic shell highlighting.